src/main/java/org/reaktivity/nukleus/http2/internal/Http2WriteScheduler.java

```diff
@@ -272,12 +272,14 @@ public boolean pushPromise(int streamId, int promisedStreamId, ListFW<HttpHeader
     }
 
     @Override
-    public boolean data(int streamId, DirectBuffer buffer, int offset, int length, IntConsumer progress)
+    public boolean data(int streamId, DirectBuffer buffer, int offset, int length)
     {
         assert length > 0;
         assert streamId != 0;
 
-        int sizeof = 9 + length;    // +9 for HTTP2 framing
+        IntConsumer progress = NOOP;
```

Comment:
Inline `progress` as `NOOP` at single call site?

Reply:
Removed progress all over as it is not used any more.